client/e2e/support/user-list.po.ts

```diff
@@ -0,0 +1,40 @@
+export class UserListPage {
+  navigateTo() {
+    return cy.visit('/users');
+  }
+
+  getUrl() {
+    return cy.url();
+  }
+
+  getUserTitle() {
+    return cy.get('.user-list-title');
+  }
+
+  getUserCards() {
+    return cy.get('.user-cards-container app-user-card');
+  }
+
+  getUserListItems() {
+    // return element(by.className('user-nav-list')).all(by.className('user-list-item'));
```

Comment:
These commented out lines (another one just below) should presumably be deleted. Or, if they serve a purpose, that should be explained in a comment.